grass7/imagery/i.landsat/i.landsat.import/i.landsat.import.py

```diff
@@ -108,26 +114,30 @@
 from grass.exceptions import CalledModuleError
 
 
-def get_files_paths(inputdir):
+def _untar(inputdir, untardir):
 
     if not os.path.exists(inputdir):
         gs.fatal(_("Input directory <{}> does not exist").format(inputdir))
 
+    if untardir is None or untardir == "":
+        untardir = inputdir
+
+    if not os.path.exists(untardir):
+        gs.fatal(_("Directory <{}> does not exist").format(untardir))
+
```

Comment:
```suggestion
        
    else:
        if not os.path.exists(untardir):
            gs.fatal(_("Directory <{}> does not exist").format(untardir))
        if not os.is_dir(untardir):
           gs.fatal(_("<{}> is not a directory").format(inputdir))
        elif not os.access(untardir, os.W_OK):
            gs.fatal(_("Directory <{}> is not writable.").format(inputdir))
```

Not sure which checks `G_M_OPT_DIR` perform by default, but if you test for existence you should probably also check that untardir is a direcory and writable. It would probably be more elegant to place that check in a function.